### ISSUE TITLE
Results views should display the number of agencies/results

### ISSUE THREAD

**Issue by Abbe98** [govdirectory/website#68]
This might be very useful when we introduce various types of interactive filters like in #64 or it might be covered by us introducing some type of stats/visualizations. 

Tech wise it's anyway very simple.

**Comment by Ainali** [comment]
Yes, something like "Displaying 143 out of 435 agencies" would be very helpful.
